disnake/http.py

```diff
@@ -171,7 +171,7 @@ def to_multipart_with_attachments(
 
 
 class Route:
```

Comment:
I have a suggestion. What if we create a constant for the API version? This will reduce the effort of switching the API version in the future.
```py
API_VERSION: int = 10

...

BASE = f"https://discord.com/api/v{API_VERSION}"
```

Reply:
I was thinking about this, but changing that constant would not change the current api version, you would still need to modify `BASE` 

Additionally, the gateway version may not match the http version, so I want to keep them separate.

Reply:
We can have 2 variables, 1 for http and 1 for gateway.

> you would still need to modify BASE

Wdym? An f-string should do the job, no?

Reply:
It would not update the constant during runtime, which I don't want anyone to believe.

Reply:
You misunderstood me. I don't want to make a constant that updates at runtime. I just want to reduce the amount of duplicated code by replacing "...v10" with f"...{API_VERSION}"